.github/renovate.json

```diff
@@ -70,6 +70,11 @@
       "matchPackagePrefixes": ["org.jacoco"],
       "allowedVersions": "!/0.8.9/"
     },
+    {
+      "matchManagers": ["maven"],
+      "matchPackagePrefixes": ["io.netty"],
+      "allowedVersions": "<=4.1.110.Final"
```

Comment:
Should work https://docs.renovatebot.com/configuration-options/#allowedversions

You'd also have to backport this to `stable/**` branches for full effect since PRs are open there as well: https://github.com/camunda/camunda/pull/19258

Alternatively you can always manually close such a PR to ignore that specific version, instead of adding a Renovate rule

Reply:
The PRs are blocked on stable branches explicitly, so I think it's fine not to backport for now.